tensormap-client/src/components/DragAndDropCanvas/Canvas.css

```diff
@@ -0,0 +1,65 @@
+.dndflow {
+    flex-direction: column;
+    display: flex;
+    flex-grow: 1;
+    height: 100%;
+  }
+  
+  .dndflow aside {
+    border-right: 1px solid #eee;
+    padding: 15px 10px;
+    font-size: 12px;
+    background: #141414;
+    color: #eee;
```

Comment:
how about we use here CSS Variables? Curious to know why we are hard-coding the values of colors?